kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md

```diff
@@ -382,23 +382,23 @@ No one should undertake a vrata in the midst of _aśauca_ (impurity on birth and
 
-[^118]: सर्वभूतभयं व्याधिः प्रमादो गुरुशासनम् । अवतमानि पश्यन्ते सकदेतानि शास्त्रतः॥ g. by कल्पतर on व्रत p. 5, हे. (on व्रत) vol, I. p. 334, व. का. वि. p. 9, ए. त, pp. 8-9 (from देवल).
-119. यत्त सत्यवतः-प्रारब्धदीर्घतपसां नारीणां यदजो भवेत् । न तत्रापि व्रतस्य स्यादुपरोधः कथंचनेति-तत्प्रतिनिधिना कारयेदित्येतत्परम् । तदुक्तं मदनरत्ने मात्स्ये-अन्तरा हरजोयोगे पूजामन्येन कारयेदिति । नि. सि. p. 29; गर्भिणी सूतिका नक्तं कुमारी च रजस्वला । पदाऽद्धा तदान्येन कारयेत् क्रियते सदा ॥ मत्स्यपुराण q. by व. का. वि. p. 10, व्रततत्व p. 152; अग्नि. (175. 39-40) has the same verse but reads कारयेत क्रियाः सदा. The printed मत्स्य 62. 34 reads गर्भिणी... कुमारी वाथ रोगिणी। यद्ययुद्धा... कारये स्मयता स्वयम् ॥. The गरुडपुराण I. 128.18 is: प्रारब्धतपसां ब्रीणां रजो...कायिकं स्वयमेव वा. ब. का. वि. explains अशुद्धा पूजादिकं कारयेत्। कायिकमुपवासादिकं सदा शुदया अशुद्धया वा स्वयं क्रियते । .. 120. अष्टो तान्यत्रतध्नानि आपो मूलं फलं पयः। इविळाणकाम्या च गुरोर्वचन मौषधम् ॥ उद्योगपर्ष 39. 71-72, अग्निपु. 175. 43, स्कन्द (acc. to हे. व्रत I. p. 333) and विष्णुरहस्य (acc. to हे. व्रतखण्ड II. p. 771). 'पयःपानादीनामवतग्नवं ग्रीवाला त्यन्तपीडितव्रतविषयम्। हे. (व्रत) I, P. 333; व. का. पि. p. 10 ascribes it to बौधायन.
+[^118]: सर्वभूतभयं व्याधिः प्रमादो गुरुशासनम् । अव्रतघ्नानि पठ्यन्ते सकृदेतानि शास्त्रतः॥ q. by कल्पतरु on व्रत p. 5, हे. (on व्रत) vol, I. p. 334, व. का. वि. p. 9, ए. त, pp. 8-9 (from देवल).
+[^119]: यत्त सत्यव्रतः-प्रारब्धदीर्घतपसां नारीणां यद्रजो भवेत् । न तत्रापि व्रतस्य स्यादुपरोधः कथंचनेति-तत्प्रतिनिधिना कारयेदित्येतत्परम् । तदुक्तं मदनरत्ने मात्स्ये-अन्तरा तु रजोयोगे पूजामन्येन कारयेदिति । नि. सि. p. 29; गर्भिणी सूतिका नक्तं कुमारी च रजस्वला । पदाऽशुद्धा तदान्येन कारयेत् क्रियते सदा ॥ मत्स्यपुराण q. by व्र. का. वि. p. 10, व्रततत्व p. 152; अग्नि. (175. 39-40) has the same verse but reads कारयेत क्रियाः सदा. The printed मत्स्य 62. 34 reads गर्भिणी... कुमारी वाथ रोगिणी। यद्यशुद्धा... कारयेरप्रयता स्वयम् ॥. The गरुडपुराण I. 128.18 is: प्रारब्धतपसां स्त्रीणां रजो...कायिकं स्वयमेव वा॥ व्र. का. वि. explains अशुद्धा पूजादिकं कारयेत्। कायिकमुपवासादिकं सदा शुद्धया अशुद्धया वा स्वयं क्रियते ।
+[^120]: अष्टो तान्यव्रतघ्नानि आपो मूलं फलं पयः। हविर्ब्राह्मणकाम्या च गुरोर्वचनमौषधम् ॥ उद्योगपर्व 39. 71-72, अग्निपु. 175. 43, स्कन्द (acc. to हे. व्रत I. p. 333) and विष्णुरहस्य (acc. to हे. व्रतखण्ड II. p. 771). 'पयःपानादीनामव्रतघ्नत्वं स्त्रीबालात्यन्तपीडितव्रतविषयम्।' हे. (व्रत) I, P. 333; व्र. का.वि. p. 10 ascribes it to बौधायन.
 
-121. गात्राम्य शिरोम्यङ्गं ताम्बूलं चानुलेपनम् । व्रतस्थो वर्जयेत्सर्व यश्चापदलरामद। q. by हे.(on व्रत) vol. I. p. 1009, व. का. वि. p. 10 (from the मिताक्षरा).
+[^121]: गात्राम्यङ्गं  शिरोम्यङ्गं ताम्बूलं चानुलेपनम् । व्रतस्थो वर्जयेत्सर्वे यच्चान्यद्वलरामकृत्। q. by हे.(on व्रत) vol. I. p. 1009, व्र. का. वि. p. 10 (from the मिताक्षरा).
```

Comment:
गात्राभ्यङ्गं शोरोभ्यङ्गं
यच्चान्यद्बलरामकृत्

Reply:
corrected. [^121]: गात्राभ्यङ्गं  शिरोभ्यङ्गं ताम्बूलं चानुलेपनम् । व्रतस्थो वर्जयेत्सर्वे यच्चान्यद्बवलरामकृत्। q. by हे.(on व्रत) vol. I. p. 1009, व्र. का. वि. p. 10 (from the मिताक्षरा).